### ISSUE TITLE
Can't be used with remote hosts

### ISSUE THREAD

**Issue by frsantos** [brentd/xray-rails#67]
When the Rails host is not the same as the developer host, there is no way to open the editor by clicking on a template. Cases when this happen are, for example, when the Rails application is run inside Docker or Vagrant.

Ideally, it should use open-in-editor URLs like [BetterError](https://github.com/charliesome/better_errors/blob/f8aa9afebe559b1148bd30d5a152dafbdd87ac61/lib/better_errors.rb#L63) or [Rails footNotes](https://github.com/josevalim/rails-footnotes#editor-links) to open the editor from the client side, instead of from the server side.

**Comment by mattbrictson** [comment]
Good idea, but this is a pretty significant change to how xray-rails is architected. We'll need some outside help to make this happen. Is anyone interested in putting together a PR?
